### PR TITLE
Card Component

### PR DESCRIPTION
**Pull request by anupkrbid**
Card Component Implementation

### CODE REVIEW

File: docs/cards.html [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+              style="
+                width: 637px;
+                height: 272px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
```
Comment (bishnoimukesh) [inline docs/cards.html]: avoid inline styling

File: docs/cards.html [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+              style="
+                width: 745px;
+                height: 416px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
```
Comment (bishnoimukesh) [inline docs/cards.html]: same

File: docs/cards.html [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+            </iframe>
```
Comment (bishnoimukesh) [inline docs/cards.html]: same

File: docs/cards.html [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+              style="
+                width: 745px;
+                height: 440px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
```
Comment (bishnoimukesh) [inline docs/cards.html]: same

File: docs/cards.html [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+              style="
+                width: 788px;
+                height: 440px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
```
Comment (bishnoimukesh) [inline docs/cards.html]: same

**Review by bishnoimukesh — COMMENTED** [review]
take care of the suggestions rest everything is fine to go

File: docs/cards.html (hunk 1 above)
Comment (anupkrbid) [reply docs/cards.html]: bro this is coming from the carbon, sh its self, each has different width based on the code i pasted

Comment (anupkrbid) [reply docs/cards.html]: fixed

**Comment by anupkrbid** [comment]
Card Component

**Pull request merged**